projects/account/src/app/modules/profile/components/cards/login/login.component.ts

```diff
@@ -61,24 +59,37 @@ export class LoginComponent {
         );
     }
 
-    updatePassword (newPassword) {
+    updatePassword (newPassword): void {
         this.profileService.changePassword(newPassword).subscribe({
-            next: () => { this.openSuccessSnackbar(); },
-            error: () => { this.openErrorSnackbar(); }
+            next: () => { this.openSnackbar('success', 'Password successfully changed'); },
+            error: () => { this.openSnackbar('error', 'An error occurred changing the password'); }
         });
     }
 
-    openErrorSnackbar() {
-        const config = new MatSnackBarConfig();
-        config.duration = fiveSeconds;
-        config.data = {type: 'error', message: 'An error occurred changing the password'};
-        this.snackbar.openFromComponent(SnackbarComponent, config);
+    openEmailAddressChangeDialog() {
+        const dialogConfig = new MatDialogConfig();
+        dialogConfig.disableClose = true;
+        dialogConfig.restoreFocus = true;
+        const dialogRef = this.changeEmailDialog.open(ChangeEmailComponent, dialogConfig);
+        dialogRef.afterClosed().subscribe(
+            (newEmailAddress) => {
+                if (newEmailAddress) {
+                    this.updateEmailAddress(newEmailAddress);
+                }}
+        );
+    }
+
+    updateEmailAddress (newEmailAddress): void {
+        this.profileService.changeEmailAddress(newEmailAddress).subscribe({
+            next: () => { this.openSnackbar('info', 'Check your inbox for a link to verify the new email address'); },
+            error: () => { this.openSnackbar('error', 'An error occurred changing the email address'); }
```

Comment:
Any idea what these errors might be, or is this a bit of a catch all?

Reply:
Don't know of anything specific that might go wrong.  It's a catch all until more specific errors are identified and can be added.